jsonschema-validation.xml

```diff
@@ -536,84 +530,66 @@
+                    for custom meta-schemas that defines "format" as an assertion. The URI for the
+                    Format-Assertion vocabulary, is:
+                    &lt;https://json-schema.org/draft/2020-11/vocab/format-assertation&gt;.
                 </t>
                 <t>
                     The current URI for the corresponding meta-schema is:
-                    <eref target="https://json-schema.org/draft/2020-11/meta/format"/>.
+                    <eref target="https://json-schema.org/draft/2020-11/meta/format"/>. Because the
+                    syntactic requirements of "format" do not change between the annotation and assertion
+                    vocabularies, the meta-schema is shared between them.
+                </t>
+                <t>
+                    Specifying both the Format-Annotation and the Format-Assertion vocabularies is functionally
+                    equivalent to specifying only the Format-Assertion vocabulary since its requirements
+                    are a superset of the Format-Annotation vocabulary.
                 </t>
-
             </section>
 
             <section title="Implementation Requirements">
                 <t>
-                    The "format" keyword functions as an annotation, and optionally as an assertion.
-                    <cref>This is due to the keyword's history, and is not in line with current
-                    keyword design principles.</cref>  In order to manage this ambiguity, the
-                    "format" keyword is defined in its own separate vocabulary, as noted above.
-                    The true or false value of the vocabulary declaration governs the implementation
-                    requirements necessary to process a schema that uses "format", and the
-                    behaviors on which schema authors can rely.
+                    The "format" keyword functions as defined by the vocabulary which is referenced.
                 </t>
 
-                <section title="As an annotation">
+                <section title="Format-Annotation Vocabulary">
                     <t>
                         The value of format MUST be collected as an annotation, if the implementation
-                        supports annotation collection.  This enables application-level validation when
+                        supports annotation collection. This enables application-level validation when
                         schema validation is unavailable or inadequate.
                     </t>
                     <t>
-                        This requirement is not affected by the boolean value of the vocabulary
-                        declaration, nor by the configuration of "format"'s assertion
-                        behavior described in the next section.
+                        Implementations MAY still treat "format" as an assertion in addition to an
```

Comment:
I was thinking we would do away with that now, so any "best-effort" validation is not our concern, and in stead is  the result of processing annotation results

Reply:
I think I'm with Henry in that it bears mentioning simply due to its history.

Reply:
OK then